Quality-Assistance.md

```diff
@@ -0,0 +1,96 @@
+- I can comfortably set up a build and delivery pipeline for a real-life application.
+- I am actively building good relationships with people in other teams like support, customer advisory, trust and security, and customer success.
+
+## I am an effective and efficient quality coach
+
+- I am well-connected to our product delivery teams, including product managers, product designers, and software engineers.
+- I can comfortably and effectively coach multiple teams in the same cycle, perhaps across different parts of the business domain.
+- I confidently help teams increase the flow of value to our customers, where work flows forward and rarely comes back due to quality concerns we missed on the first pass.
+- I confidently partner with teams using an iterative development approach, from walking skeleton to polished product.
+- I confidently partner with teams through building smaller improvements within a single cycle, or complex value streams spanned across multiple cycles.
+- I confidently help product teams make pragmatic decisions during the fixed timelines of our build cycles, taking the bigger picture into account.
+- I confidently organise, coordinate, and run quality coaching activities, like QA KickOff and QA Demo, and how to reap the most value from these activities.
+
+# Lead Quality Coach (Level 4)
+
+> Building on Senior Quality Coach (Level 3), I am a force multiplier, deliberately shifting my focus from individual quality coaching toward uplifting the whole product delivery team.
+
+## I am an expert in our customers and our business and could be an effective product manager or customer advocate
+
+- I have a broad and thorough understanding of the problems our customers are trying to solve and how we do/don't solve them, helping me make good tradeoffs with the entire picture in mind.
+- In addition to our product delivery teams, I am well-connected to our other teams, like support, customer advisory, trust and security, and customer success.
+- I have attained mastery in many customer problem+solution areas and am regularly helping others attain mastery themselves.
+- People genuinely want my help shaping new ideas, to explore new customer problems+solutions and discover potential risks.
+
+## I am actively improving our entire product delivery team
+
+- I have a clear picture of the ideal product delivery team and help our teams make appropriate and timely improvements each cycle.
+- I help our team clearly identify negative quality trends and help make appropriate and timely improvements each cycle.
+- I raise appropriate and timely concerns in #product-delivery and contribute effectively to conversations that help clarify ways we can improve.
+
+## I am actively improving our quality coaching team
+
+- I have a clear picture of the ideal quality coaching team and help make appropriate and timely improvements each cycle.
```

Comment:
Elaborating on 
`help make appropriate and timely improvements each cycle.`
- I experiment with different quality coaching activities to identify the practices that will provide the most confidence in delivery each cycle.

Reply:
```suggestion
- I help the quality coaching team improve how they do daily work so we can achieve better outcomes with the teams we coach.
```